### PR TITLE
Introduction fixes

### PR DESCRIPTION
**Pull request by tmciver**
Fixed some whitespacing, typos and broken image links.

### CODE REVIEW

**Comment by adamwespiser** [comment]
Thank you for working on this! I really like the bulleted questions and will likely push this throughout the chapters.

File: docs/01_introduction.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ This may seem like a lot.  But don't worry, all these things, and more, are alre
 
 ## Project Road Map: What do we have?
 
-![](../img/WYAS-Dependency-Tree.png)
+![](./img/WYAS-Dependency-Tree.png)
```
Comment (tmciver) [inline docs/01_introduction.md]: @adamwespiser Why this change?  It doesn't seem to be the correct path and the images do not show up when viewing the markdown in github.

Comment (adamwespiser) [inline docs/01_introduction.md]: whoops, forgot to include this: I'm rendering the pages using Hakyll in a different project that manages my website. The resources (img/pdf/jpeg, etc) location are all aggregated together in a single location, different from this.

Comment (tmciver) [inline docs/01_introduction.md]: I thought it might have been something like that.  It's a shame; it would be nice if it worked on Github too.  Is it possible to make it work for both?
